cmd/cloud/installation_flag.go

```diff
@@ -209,6 +210,126 @@ func (flags *installationDeleteFlags) addFlags(command *cobra.Command) {
 	_ = command.MarkFlagRequired("installation")
 }
 
+type baseVolumeFlags struct {
+	mountPath string
+	readOnly  bool
+	filename  string
+	data      string
+}
+type installationCreateVolumeFlags struct {
+	clusterFlags
+	baseVolumeFlags
+	installationID string
+	volumeName     string
+}
+
+func (flags *installationCreateVolumeFlags) addFlags(command *cobra.Command) {
+	command.Flags().StringVar(&flags.installationID, "installation", "", "The id of the installation to create a volume for.")
+	command.Flags().StringVar(&flags.volumeName, "volume-name", "", "The name of the volume to create.")
+	command.Flags().StringVar(&flags.mountPath, "mount-path", "", "The container path to mount the volume in.")
+	command.Flags().BoolVar(&flags.readOnly, "read-only", true, "Whether the volume should be read only or not.")
```

Comment:
Do we need a read-only flag as a mount of type Secret is always read-only https://kubernetes.io/docs/concepts/storage/volumes/#secret

Reply:
Good catch, but this is just prepping for the future when we may have other volume types.